.github/workflows/xdist_testing.yml

```diff
@@ -39,4 +39,8 @@ jobs:
       run: |
         source /opt/conda/bin/activate mssenv \
         && cd $GITHUB_WORKSPACE \
-        && pytest -n 6 --dist loadscope --max-worker-restart 0 mslib
+        && pytest -n 6 --dist loadscope --max-worker-restart 0 mslib \
+        || (for i in {1..5} \
+          ; do pytest mslib --last-failed --lfnf=none \
```

Comment:
this means the fallback is to run serial tests? because there is not -n option given.

Reply:
xdist now runs the reruns in parallel also.